src/signals/incident/definitions/__tests__/wizard-step-2-vulaan.test.js

```diff
@@ -14,6 +14,7 @@ const defaultControls = {
   error: expect.objectContaining({}),
   custom_text: expect.objectContaining({}),
   $field_0: expect.objectContaining({}),
+  help_text: expect.objectContaining({}),
```

Comment:
This isn't much of a test. Would make more sense to test for
```javascript
expect.objectContaining({
    meta: {
      label: configuration.language.helpTextHeader,
      value: configuration.language.helpText,
    },
})
```

Reply:
👍